clone.py

```diff
@@ -72,9 +78,11 @@ def main():
             raise ExitException(1, "'version' must be a single argument. "
                                 "See usage with '-h'.")
         if options.mp_only:
-            repo.clone_mp(options.marketplace_conf if options.marketplace_conf else '', options.fallback_branch)
+            repo.clone_mp(options.marketplace_conf if options.marketplace_conf else '', options.fallback_branch,
+                          options.depth, options.unshallow)
```

Comment:
-1 for adding two low level options here (and in all sub-methods), we may want to pass a lot of other Git options. Why those two only?
What about putting those in the `marketplace_conf`?
Look at creating a GitOptions object or a defaults map in `__init__` method or any other pythonic solution.

Reply:
The codebase on clone/fetch hasn't really changed since Dec. 2013, I guess it already fits our need, so no need to open to all git options IMHO.
Those are new options so we can tune the clone for optimizing the required time.
As such, they should also be only set when calling the release script but not globally on the marketplace file.

Reply:
That's an internal detail and can perfectly be refactored later if you wish. What's important now is the feature.